docker/launch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+trap ctrl_c INT
+
+function ctrl_c() {
+    docker-compose down
```

Comment:
Are you sure that `docker-compose down` isn't the same thing like just sending `SIGINT` to the `docker-compose up` process? This looks like a single `ctrl+c` should result in the same actually: https://github.com/docker/compose/blob/599b29e405164c42168296f60aa23b0ef6c062f0/compose/cli/main.py#L1053